sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeInvoicesOptions.cs

```diff
@@ -33,8 +33,8 @@ public RecognizeInvoicesOptions()
         public FormContentType? ContentType { get; set; }
 
         /// <summary>
-        /// Locale value. Supported locales include: en-US.
```

Comment:
would it make sense to keep this info? I know the user can look at options in the enum, but since not all those options are supported for invoice this sentence could be helpful.

Reply:
I was thinking about it, but every time the service makes an update to the supported locales I will have to come here an update the docstring...
I will bring it back for now and then we'll see